### PR TITLE
Implement `SeamWebhook` class for handling svix webhooks

### PR DESCRIPTION
**Pull request by andrii-balitskyi**
Closes https://github.com/seamapi/python-next/issues/27

### CODE REVIEW

**Review by razor-x — COMMENTED** [review]
Nice! We need to "export" this at the top level and add the corresponding section to the readme

File: README.rst [hunk 1]
```diff
@@ -209,6 +209,45 @@ A Personal Access Token is scoped to a Seam Console user. Obtain one from the Se
...
+      payload = request.get_data()
+
+      try:
+          data = webhook.verify(payload, headers)
+      except Exception:
+          return jsonify({"error": "Invalid payload"}), 400
+
+      error = store_event(data)
+      if error:
+          return jsonify({"error": "Failed to store event"}), 500
```
Comment (razor-x) [inline README.rst:243]:
```suggestion
      try:
          data = webhook.verify(payload, headers)
      except Exception:
          return 'Bad Request', 400

      try:
          store_event(data)
      except Exception:
          return 'Internal Server Error', 500
```

File: README.rst [hunk 2]
```diff
@@ -209,6 +209,45 @@ A Personal Access Token is scoped to a Seam Console user. Obtain one from the Se
...
+  webhook_secret = os.getenv('SEAM_WEBHOOK_SECRET')
+  webhook = SeamWebhook(webhook_secret)
+
+  @app.route('/webhook', methods=['POST'])
+  def handle_webhook():
+      headers = request.headers
+      payload = request.get_data()
+
+      try:
+          data = webhook.verify(payload, headers)
```
Comment (razor-x) [inline README.rst:237]:
```suggestion
      try:
          data = webhook.verify(request.get_data(), request.headers)
```

File: README.rst [hunk 3]
```diff
@@ -209,6 +209,45 @@ A Personal Access Token is scoped to a Seam Console user. Obtain one from the Se
...
   workspaces = seam.workspaces.list()
 
+Webhooks
+~~~~~~~~
+
+The Seam API implements webhooks using `Svix <https://www.svix.com>`_. This SDK exports a thin wrapper ``SeamWebhook`` around the svix package. Use it to parse and validate Seam webhook events.
+
+Refer to the `Svix docs on Consuming Webhooks <https://docs.svix.com/receiving/introduction>`_ for an in-depth guide on best-practices for handling webhooks in your application.
+
+.. code-block:: python
```
Comment (razor-x) [inline README.rst:219]: Not sure if you did this, but let's run it through https://black.vercel.app/ to check syntax

File: README.rst [hunk 4]
```diff
@@ -209,6 +209,45 @@ A Personal Access Token is scoped to a Seam Console user. Obtain one from the Se
...
+
+  import os
+  from flask import Flask, request, jsonify
+  from seam import SeamWebhook
+  from .store_event import store_event
```
Comment (razor-x) [inline README.rst:224]:
```suggestion
  import os

  from flask import Flask, request, jsonify
  from seam import SeamWebhook

  from .store_event import store_event
```

**Review by razor-x — APPROVED** [review]

File: README.rst [hunk 5]
```diff
@@ -209,6 +209,45 @@ A Personal Access Token is scoped to a Seam Console user. Obtain one from the Se
...
+          return jsonify({"error": "Invalid payload"}), 400
+
+      error = store_event(data)
+      if error:
+          return jsonify({"error": "Failed to store event"}), 500
+
+      return '', 204
+
+  if __name__ == '__main__':
+      app.run(port=5000)
```
Comment (razor-x) [inline README.rst:248]: I love that this is a self contained example that just runs! Can we verify the server starts before merging?

Comment (razor-x) [reply README.rst:248]: I updated the JS example so it "just works" (as long as you install the npm libs): https://github.com/seamapi/javascript-webhook/pull/9/files

Comment (razor-x) [reply README.rst:248]: Can we update this so it "just works" too?

Comment (razor-x) [inline README.rst:248]:
```suggestion
  import os

  from flask import Flask, request
  from seam import SeamWebhook

  app = Flask(__name__)

  webhook = SeamWebhook(os.getenv('SEAM_WEBHOOK_SECRET'))

  @app.route('/webhook', methods=['POST'])
  def handle_webhook():
      try:
          data = webhook.verify(request.get_data(), request.headers)
      except Exception:
          return 'Bad Request', 400

      try:
          store_event(data)
      except Exception:
            return 'Internal Server Error', 500

      return '', 204

  def store_event(data):
      print(data)

  if __name__ == '__main__':
      app.run(port=8080)
```

Comment (razor-x) [reply README.rst:248]: I tested this locally and suggested it here: https://github.com/seamapi/python-next/pull/60/files#r1609088762

Comment (andrii-balitskyi) [reply README.rst]: Cool! Thank you

**Pull request merged**